src/FtlPlayer.ts

```diff
@@ -1,15 +1,44 @@
 import { Janus, PluginHandle, Message, JSEP } from "janus-gateway";
 
+type Options = {
+    debug?: boolean,
+    hideMediaControls?: boolean
+}
+
 export class FtlPlayer {
     private static readonly janusPluginPackage = "janus.plugin.ftl";
+    private readonly logPrefix = "janus-ftl-player:";
     private element: HTMLVideoElement;
     private serverUri: string;
     static isJanusInitialized: boolean = false;
     private janusInstance: (Janus | null) = null;
     private janusPluginHandle: (PluginHandle | null) = null;
+    private options: Options;
+
+    constructor(element: HTMLVideoElement, serverUri: (string | null) = null, options: Options) {
+        // Allow configuring additional options
+        this.options = {
+            debug: true,
+            hideMediaControls: true
+        };
+        Object.assign(this.options, options);
 
-    constructor(element: HTMLVideoElement, serverUri: (string | null) = null) {
         this.element = element;
+
+        if (this.options.hideMediaControls) {
+            let uniqueClass = "janus-ftl-player-" + Math.floor(Math.random() * 10000)
+            this.element.classList.add(uniqueClass);
+
+            let videoStyle = document.createElement("style");
+            videoStyle.appendChild(document.createTextNode(`
+            .${uniqueClass}::-webkit-media-controls-timeline,
+            .${uniqueClass}::-webkit-media-controls-timeline {
```

Comment:
Are the duplicate selectors needed?

Reply:
My mistake, copied the code we wrote for video & audio.